posthog/migrations/0896_datawarehousesavedquery_source.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.22 on 2025-10-29 08:15
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0895_savedheatmap_heatmapsnapshot_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="datawarehousesavedquery",
+            name="source",
+            field=models.CharField(
+                blank=True,
+                choices=[("root", "Root"), ("endpoint", "Endpoint"), ("revenue_analytics", "Revenue Analytics")],
+                default=None,
+                help_text="Where this SavedQuery is created.",
+                null=True,
+            ),
+        ),
+    ]
```

Comment:
I see there's another PR for populating this - so will leave the backfill for then